clang/lib/Driver/SanitizerArgs.cpp

```diff
@@ -1275,13 +1275,23 @@ void SanitizerArgs::addArgs(const ToolChain &TC, const llvm::opt::ArgList &Args,
 
       CmdArgs.push_back("-mllvm");
       CmdArgs.push_back("-msan-eager-checks=1");
+    } else if (Sanitizers.has(SanitizerKind::Thread)) {
```

Comment:
Please add a descriptive comment for all the newly added code.

Reply:
Done